applications/openshift/registry/imagestream_sets_schedule/rule.yml

```diff
@@ -42,22 +42,20 @@ ocil: |-
   <pre>oc get imagestream  --all-namespaces -o jsonpath='{{{ jqfilter }}}'</pre>
   Alternatively, to view a list of ImageStreams that do not schedule updates,
   run:
-  <pre>oc get imagestreams -A -ojson | jq '.items[] | select(.spec.tags[]?.importPolicy.scheduled != true) | .metadata.name' | sort | uniq</pre>
+  <pre>oc get imagestreams -A -ojson | jq -r '.items[] | select(.spec.tags[]?.importPolicy.scheduled != true) | "\(.metadata.namespace),\(.metadata.name)"' | sort | uniq</pre>
```

Comment:
Wondering if we also need `select(.from.kind != "ImageStreamTag")` here

Reply:
Sure - I can add that. will take care of it today